tensorflow/core/kernels/mkl_tfconv_op.cc

```diff
@@ -29,8 +29,8 @@ limitations under the License.
 #include "tensorflow/core/util/tensor_format.h"
 
 #include "tensorflow/core/util/mkl_util.h"
-#include "third_party/mkl/include/mkl_dnn.h"
-#include "third_party/mkl/include/mkl_dnn_types.h"
+#include "mkl_dnn.h"
```

Comment:
Don't do that. This will cause various issues. Always use full include path in tensorflow code please.